src/CloudFoundry.VisualStudio/CloudFoundry.VisualStudioPackage.cs

```diff
@@ -154,11 +154,11 @@ private void ButtonBuildAndPushProjectExecuteHandler(object sender, EventArgs e)
             {
                 PushEnvironment environment = new PushEnvironment();
                 environment.ProfileFilePath = Path.Combine(VsUtils.GetPublishProfilePath(),
-                    string.Format(CultureInfo.InvariantCulture, "{0}.cf.pubxml", PushEnvironment.DefaultProfileName));
+                    string.Format(CultureInfo.InvariantCulture, "{0}{1}", PushEnvironment.DefaultProfileName, PushEnvironment.Extension));
```

Comment:
this line of code is superfluous - it duplicates what the constructor of PushEnvironment does afaik